x-pack/legacy/plugins/uptime/common/runtime_types/ping/ping.ts

```diff
@@ -0,0 +1,169 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import * as t from 'io-ts';
+
+export const HttpResponseBodyType = t.partial({
+  bytes: t.number,
+  content: t.string,
+  content_bytes: t.number,
+  hash: t.string,
+});
+
+export type HttpResponseBody = t.TypeOf<typeof HttpResponseBodyType>;
+
+export const TlsType = t.partial({
+  certificate_not_valid_after: t.string,
+  certificate_not_valid_before: t.string,
+});
+
+export type Tls = t.TypeOf<typeof TlsType>;
+
+export const PingType = t.intersection([
+  t.type({
+    '@timestamp': t.string,
```

Comment:
i think @timestamp can't be accessed using a dot notation from object, so maybe we should continue using timestamp.

Reply:
We can change it to not use `@`, but we will want to drop the `@timestamp` and use mapping code everywhere in the app. We previously had both being returned; I talked to @andrewvc about this and we decided to keep the `@` because it meant less mapping code, but I also see the convenience of what you're proposing.

I am fine with changing it to drop the `@`.

Reply:
I'm OK with dropping the @ as long as we are consistent

Reply:
Ok, sounds good. I'll update the new type and that should take care of it.

Reply:
Done https://github.com/elastic/kibana/pull/59392/commits/1062e415543f73129b43091db7fe1df43bcbefc5.